File: py/day2/day2_test.py
from day2 import product_of_all_other

def test_day2_1():
    assert product_of_all_other([1, 2, 3, 4, 5]) == [120, 60, 40, 30, 24]
    
def test_day2_2():
    assert product_of_all_other([]) == []
    
def test_day2_3():
    assert product_of_all_other([1, 2, 3]) == [6,3,2]

File: js/day2/day2.test.js
const productOfAllOther = require("./day2");
describe("Day2", () => {
  test("given [1, 2, 3, 4, 5] return [120, 60, 40, 30, 24]", () => {
    expect(productOfAllOther([1, 2, 3, 4, 5], 17)).toEqual([120, 60, 40, 30, 24]);
  });
});
